frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -51,6 +59,24 @@ export const FormComponentConfig = ({
 
   const unsupportedPropertyKeys: string[] = getUnsupportedPropertyTypes(rest);
 
+  const booleanRestPropertyKeys = getFilteredPropertyKeys((propertyKey) => {
+    return (
+      rest[propertyKey]?.type === 'boolean' ||
+      rest[propertyKey]?.$ref?.endsWith(getExpressionSchemaDefinitionReference('boolean'))
+    );
+  }, rest);
+
+  const objectRestPropertyKeys = getFilteredPropertyKeys((propertyKey) => {
+    return rest[propertyKey]?.type === 'object' && rest[propertyKey]?.properties;
+  }, rest);
+
+  const remainingRestPropertyKeys = Object.keys(rest).filter((propertyKey) => {
+    return (
+      !booleanRestPropertyKeys.includes(propertyKey) &&
+      !objectRestPropertyKeys.includes(propertyKey)
+    );
+  });
+
```

Comment:
Could we extract the matcher functions and give them some more meaningfull names so these consts would be easier to read?